parser-typechecker/src/Unison/Codebase/Editor/Git.hs

```diff
@@ -122,29 +123,30 @@ importRemoteBranch
   :: forall m v a
    . MonadIO m
   => Codebase m v a
+  -> Branch.Cache m
   -> RemoteNamespace
   -> ExceptT GitError m (Branch m)
-importRemoteBranch codebase ns = do
-  (branch, cacheDir) <- viewRemoteBranch' ns
+importRemoteBranch codebase cache ns = do
+  (branch, cacheDir) <- viewRemoteBranch' cache ns
```

Comment:
This file had a bunch of places where the cache is being threaded through. Very mechanical.